cmd/log-parser/README.md

```diff
@@ -92,7 +92,12 @@ To merge all logs:
    ```
    $ sudo chown $USER *.log
    ```
-1. Run the script:
+1. To install the program:
+   ```
+   $ go get -d github.com/kata-containers/tests
+   $ pushd . && cd $GOPATH/src/github.com/kata-containers/tests/cmd/log-parser && make install && popd
```

Comment:
why not just `pushd` directly to the destination path, rather than involve `.`? Maybe there is an idiom here I don't know of?
```bash
pushd $GOPATH/src/github.com/kata-containers/tests/cmd/log-parser && make install && popd
```

Reply:
@grahamwhaley : fixed!